README.md

```diff
@@ -67,3 +67,9 @@ auth:
     keyPath: ~/.spin/keyfile
     certPath: ~/.spin/certfile
 ```
+
+# Usage
+
+Download the `spin` binary from a [release](https://github.com/spinnaker/spin/releases) and
```

Comment:
How is the download done? Can we provide an easy curl command?

Reply:
Is there a latest release url for github?

Reply:
I guess we could set a `$VERSION` and then give a curl command.